x/cert/keeper/grpc_query.go

```diff
@@ -75,6 +75,10 @@ func (q Querier) Certificates(c context.Context, req *types.QueryCertificatesReq
 				limit = uint(req.Pagination.Limit)
 			}
 
+			if limit == 0 && offset == 0 {
+				offset = 1
```

Comment:
Default offset should be 0. May be we can set default limit but I believe there's a default limit from the SDK already

Reply:
if `offset` is 0 `KVStorePrefixIteratorPaginated` exits immediately